xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAE.cpp

```diff
@@ -270,10 +270,39 @@ AEAudioFormat CEngineStats::GetCurrentSinkFormat()
   return m_sinkFormat;
 }
 
+enum AE_STATES
```

Comment:
leave this where it was. I had a reason for this

Reply:
I couldn't I needed to move it above the constructor since I fixed the warning with the uninitialized value using it. If you want me to put that back I can.